app/lib/meadow_web/resolvers/data.ex

```diff
@@ -141,6 +142,28 @@ defmodule MeadowWeb.Resolvers.Data do
     end
   end
 
+  def list_ingest_bucket_objects(_, %{prefix: prefix}, _) do
+    {:ok, S3Utils.list_ingest_bucket_objects(prefix: prefix)}
+  end
+
+  def list_ingest_bucket_objects(_, _, _) do
+    {:ok, S3Utils.list_ingest_bucket_objects()}
+  end
+
+  def replace_file_set(_, %{id: id} = params, _) do
+    file_set = FileSets.get_file_set!(id)
+
+    case Pipeline.replace_the_file_set(file_set, Map.delete(params, :id)) do
+      {:error, changeset} ->
+        {:error,
+         message: "Could not replace file set",
+         details: ChangesetErrors.humanize_errors(changeset)}
+
+      {:ok, file_set} ->
+        {:ok, file_set}
+    end
+  end
+
```

Comment:
This looks like a dupe of the function below, and results in a warning:
```
warning: this clause cannot match because a previous clause at line 153 always matches
  lib/meadow_web/resolvers/data.ex:167
```

Reply:
Will fix and squash before merge once the PR is approved, thanks!